content/en/docs/concepts/services-networking/service.md

```diff
@@ -666,6 +666,22 @@ The value of `spec.loadBalancerClass` must be a label-style identifier,
 with an optional prefix such as "`internal-vip`" or "`example.com/internal-vip`".
 Unprefixed names are reserved for end-users.
 
+#### Specifying IPMode of load balancer status {#load-balancer-ip-mode}
+
+{{< feature-state for_k8s_version="v1.29" state="alpha" >}}
+
+Starting with Kubernetes 1.29 in alpha, a feature gate named `LoadBalancerIPMode` allows you to set the `.status.loadBalancer.ingress.ipMode` 
+for a Service with `type` set to `LoadBalancer`.
+`.status.loadBalancer.ingress.ipMode` specifies how the load-balancer IP behaves, and may only be specified when the `.status.loadBalancer.ingress.ip` field is specified.
+
+You can set for `.status.loadBalancer.ingress.ipMode` with two possible values "VIP" and "Proxy". 
+The default value is "VIP".
+Setting this to "VIP" indicates that traffic is delivered to the node 
+with the destination set to the load-balancer's IP and port.
+Setting this to "Proxy" indicates that traffic is delivered to the node or pod 
+with the destination set to the node's IP and node port or the pod's IP and port.
```

Comment:
Better explicitly state when the traffic is routed to the node's IP and node port,
and when it is routed to the Pod's IP and port. Maybe related to whether a NodePort
is allocated?

Reply:
@tengqm Thanks a lot for your suggestions. I've added an explicit statement for this.